docs/src/main/asciidoc/panache-rest.adoc

```diff
@@ -0,0 +1,148 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+
+= Quarkus - Generating CRUD resources with Panache REST
+
+include::./attributes.adoc[]
+:extension-status: experimental
+
+A lot of web applications are monotonous CRUD applications with boring to write REST APIs.
+To ease this task Panache REST can generate the basic CRUD endpoints for your entities and repositories.
+
+While this extension is still experimental and provides a limited feature set, we hope to get an early feedback for it.
```

Comment:
Perhaps this should be in a note?

Reply:
There is a generic note about the experimental feature already. Do you think I should add another one?

Reply:
Oh OK. If it's used elsewhere as well, let's keep it as it is